hack/virtc/README.md

```diff
@@ -75,6 +75,28 @@ I0410 08:58:49.059044    5384 proxy.go:566] proxy started
 ```
 The proxy socket specified in the example log output has to be used as `virtc`'s `--proxy-url` option.
 
+### Get cc-shim (optional)
+
+If you plan to start `virtc` with the hyperstart agent (implying the use of `cc-proxy` as a proxy), you will have the possibility to enable [cc-shim](https://github.com/clearcontainers/shim) in order to interact with the process running inside your container. First, you will have to perform extra steps to setup your environment.
+
+```
+$ go get github.com/clearcontainers/shim
+$ make
+$ sudo make install
+```
+
+The shim will be installed at the following location: `/usr/libexec/cc-shim`. There will be two cases where you will be able to enable `cc-shim`:
+
+_Start a new container_
+
+```
+# ./virtc container start -id=1 -pod-id=306ecdcf-0a6f-4a06-a03e-86a7b868ffc8 -cc-shim -cc-shim-path="/usr/libexec/cc-shim"
```

Comment:
- Although single-dash options might be allowed, I think we should stick with documenting using double-dashes (since that's what `virtc container --help` for example shows).
- Also, how about changing `-pod-id=` to:
  ```
  pod=$(uuidgen -r)
  ./hack/virtc/virtc container start -id=1 --pod-id="$pod" ...
  ```